Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md

```diff
@@ -20,6 +20,7 @@ To consume the CLI tools installed as part of the build, do the following:
 
 * Add `<repo_root>\bin\tools\cli\bin` to the path
 * set `DOTNET_HOME` environment variable to `<repo_root>\bin\tools\cli`
+* On windows ensure you are using a VS developer X64 command prompt
```

Comment:
VS has a Developer command prompt that is different from "Native Tools for X64" command prompt. We should update to reflect the exact command prompt type.